### PR TITLE
UIIN-2692: Moving items between holdings records doesn't update correctly in UI

### PR DESCRIPTION
**Pull request by mariia-aloshyna**
## Purpose

When moving items between holdings records on the same instance, the UI does not update, so the items appear in both holdings records until the instance detail pane is closed and reopened.

## Approach

- Set empty array to items component state when `items` is empty.

## Refs

https://issues.folio.org/browse/UIIN-2692

### CODE REVIEW

File: src/Instance/ItemsList/ItemsListContainer.js [hunk 1]
```diff
@@ -42,9 +42,9 @@ const ItemsListContainer = ({
   const { totalRecords } = useHoldingItemsQuery(holding.id, { searchParams: { limit: 0 }, key: 'itemCount', tenantId });
 
   useEffect(() => {
-    if (!isEmpty(items)) {
-      setItemsToShow(items);
-    }
+    const updatedItemsList = !isEmpty(items) ? items : [];
```
Comment (Terala-Priyanka) [inline src/Instance/ItemsList/ItemsListContainer.js:45]: Looks like items is always an array either with length 0 or greater than 0 if there are items in holdings.
hence, do you need this isEmpty check on items? You should probably update the state when new items are available.

Comment (mariia-aloshyna) [reply src/Instance/ItemsList/ItemsListContainer.js:45]: @Terala-Priyanka `isEmpty` also checks for `undefined` and `null`, so if `items` is undefined nothing crushes. I need to update the state even in the case when all items have been moved to another holding and it's empty.

**Comment by mariia-aloshyna** [comment]
Hi @Dmytro-Melnyshyn and @BogdanDenis please take a look

File: src/Instance/ItemsList/ItemsListContainer.js (hunk 1 above)
Comment (Terala-Priyanka) [reply src/Instance/ItemsList/ItemsListContainer.js:45]: @mariia-aloshyna 
Looks like API throws an array always. Do you see any case when a null is sent through API?
As long as it is array. we don't need to have this isEmpty check. directly update the state with new response.

Comment (mariia-aloshyna) [reply src/Instance/ItemsList/ItemsListContainer.js]: @Terala-Priyanka updated

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**